server.go

```diff
@@ -64,7 +64,12 @@ type Server struct {
 	TraceAddr   *net.UDPAddr
 	RcvbufBytes int
 
+	Interval             time.Duration
```

Comment:
I'm not sure these members need to be or should be public, but most of the members are so maybe that is consistent?